Assets/Scripts/StandardSamples/SignInWithGoogle/SignInWithGoogleManager.cs

```diff
@@ -0,0 +1,101 @@
+/*
+* Copyright (c) 2024 PlayEveryWare
+* 
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+* 
+* The above copyright notice and this permission notice shall be included in all
+* copies or substantial portions of the Software.
+* 
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+using UnityEngine;
+
+namespace PlayEveryWare.EpicOnlineServices.Samples
+{
+    class EOSCredentialManagerCallback : AndroidJavaProxy
+    {
+        public AndroidJavaObject loginObject;
+        public System.Action<string, string> callback;
+        public string name;
+        public string idToken;
+        
+        /// <summary>
+        /// Proxy class to receive Android callbacks in C#
+        /// </summary>
+        public EOSCredentialManagerCallback() : base("androidx.credentials.CredentialManagerCallback") { }
+
+        /// <summary>
+        /// Succeeding Callback of GetCredentialAsync  
+        /// GetCredentialAsync is called in com.playeveryware.googlelogin.login.SignInWithGoogle)
+        /// </summary>
+        /// <param name="credentialResponseResult"></param>
+        public void onResult(AndroidJavaObject credentialResponseResult)
```

Comment:
Since we're back in C# land, can this be `OnResult`? Or does it need to match the capitalization?

Reply:
It has to match the java interface that it's adopting.